README.md

```diff
@@ -113,12 +115,34 @@ catkin do the following:
       ## TeXlipse
       .texlipse
       ```
+
+#### Multiple Packages
+The _eclipsify_ command can be used in conbitation with the catkin command to easily generate project files for all the ROS packages that  
+reside in a workspace. The options are as follows:
+
+  1. Create project files in the catkin worspace devel directory:
+    ```
+    catkin list -u | xargs -I pkg bash -c "eclipsify pkg"
```

Comment:
This seems a bit overcomplicated to me. What about `xargs eclipsify`? That should work equivalently.